### PR TITLE
Fixed Missing TB Code Element Value

### PR DESCRIPTION
**Pull request by wonderfulguru1**
Fixed Missing TB Code Element Value

### CODE REVIEW

**Review by wandechris — APPROVED** [review]
we need to find a way to externalize these maps

File: omod/src/main/java/org/openmrs/module/nigeriaemr/ndrfactory/PharmacyDictionary.java [hunk 1]
```diff
@@ -275,7 +276,65 @@ public void loadDictionary() {
...
+        regimenCodeDescTextMap.put(71060,"Amikacin");
+        regimenCodeDescTextMap.put(78385,"Kanamycin");
+        regimenCodeDescTextMap.put(72794,"Capreomycin");
+        regimenCodeDescTextMap.put(74123,"Cycloserine");
+        regimenCodeDescTextMap.put(84360,"STREPTOMYCIN");
+        regimenCodeDescTextMap.put(82900,"PYRAZINAMIDE");
+        regimenCodeDescTextMap.put(75976,"Ethionamide");
+        regimenCodeDescTextMap.put(80133,"Moxifloxacin");
+        regimenCodeDescTextMap.put(78788,"Levofloxacin");
+        regimenCodeDescTextMap.put(72794,"CAPREOMYCIN");
```
Comment (wandechris) [inline omod/src/main/java/org/openmrs/module/nigeriaemr/ndrfactory/PharmacyDictionary.java:315]: @wonderfulguru1 there are duplicates in this map, is that intentional?

e.g 
 regimenCodeDescTextMap.put(72794,"Capreomycin");
and 
regimenCodeDescTextMap.put(72794,"CAPREOMYCIN");

**Review by wandechris — CHANGES_REQUESTED** [review]
check the duplicates

File: omod/src/main/java/org/openmrs/module/nigeriaemr/ndrfactory/PharmacyDictionary.java (hunk 1 above)
Comment (wandechris) [reply omod/src/main/java/org/openmrs/module/nigeriaemr/ndrfactory/PharmacyDictionary.java:315]: regimenCodeDescTextMap.put(71060,"AMIKACIN"); and  regimenCodeDescTextMap.put(71060,"Amikacin");
 regimenCodeDescTextMap.put(80133,"Moxifloxacin"); and regimenCodeDescTextMap.put(80133,"Moxifloxacin");

Comment (wonderfulguru1) [reply omod/src/main/java/org/openmrs/module/nigeriaemr/ndrfactory/PharmacyDictionary.java:315]: @wandechris  we will check this revert thanks

**Pull request merged**